cypress/integration/pages/onDemandRadio/index.js

```diff
@@ -40,6 +41,8 @@ Object.keys(config)
           crossPlatformTests({
             service,
             pageType,
+            variant,
+            isAmp: true,
```

Comment:
I'm probably just missing something but kinda confused by the `isAmp` prop declaration & the defaults further down. I think all of this is mirrored in the TV equivalent, but using the radio examples:

in [`crossPlatformTests` isAmp is assigned a default of false](https://github.com/bbc/simorgh/pull/7250/files#diff-03edb44bbcc5a39ca06f27af79f209baR10), but isn't it always 'true' because of this line? 

Again this prop is defaulted in the [`onDemandRadioTv` embedUrl helper](https://github.com/bbc/simorgh/pull/7250/files#diff-15eacb4558e714a81d5c04b13e53a825R15) but wouldn't it always be true? 

A prop will also default to `undefined` if not declared, which is falsey, so do you need the `= false` defaults anyway?